Formula/bwidget.rb

```diff
@@ -0,0 +1,30 @@
+class Bwidget < Formula
+  desc "Tcl/Tk script-only set of megawidgets to provide the developer additional tools"
+  homepage "https://core.tcl-lang.org/bwidget/home"
+  url "https://downloads.sourceforge.net/project/tcllib/BWidget/1.9.16/bwidget-1.9.16.tar.gz"
+  sha256 "bfe0036374b84293d23620a7f6dda86571813d0c7adfed983c1f337e5ce81ae0"
+  license "TCL"
+
+  uses_from_macos "tcl-tk" => :build
```

Comment:
```suggestion
  depends_on "tcl-tk"
```
The stock macOS Tcl doesn't know about Homebrew anyway.

Reply:
Change made with 7323f19.